Cargo.toml

```diff
@@ -488,7 +488,7 @@ sources-apache_metrics = []
 sources-aws_ecs_metrics = []
 sources-aws_kinesis_firehose = ["dep:base64", "dep:infer"]
 sources-aws_s3 = ["aws-core", "dep:aws-sdk-sqs", "dep:aws-sdk-s3", "dep:semver", "dep:async-compression", "sources-aws_sqs", "tokio-util/io"]
-sources-aws_sqs = ["aws-core", "dep:aws-sdk-sqs"]
```

Comment:
Did this turn out to be necessary? It seems like you updated the feature gates below.

Reply:
Yes? The SQS errors are still defined in the S3 source and (at least for integration tests) fail to compile.